### PR TITLE
[MAT-3798] Measure Scoring Validation

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## MADiE PR

Jira Ticket: [MAT-3798](https://jira.cms.gov/browse/MAT-3798)
(Optional) Related Tickets:

#### Validate Measure Scoring on measure create/update
Binding the permissible values of measure scoring to an enum. @sb-cecilialiu added an enum validator annotation that compares the de-serialized String against the values of the specified enum. The serialized value of the enum can be obtained using the built-in `toString()`.

- Run measure scoring validators after other validations to collect all issues
- Add periods to error messages.
- Add measuresScoring to test data.
- Add `getText()` method to retrieve the human friendly text representation of the measure scoring enum.

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by gregory-akins — APPROVED** [review]

File: src/test/java/cms/gov/madie/measure/resources/MeasureControllerMvcTest.java [hunk 1]
```diff
@@ -372,20 +389,6 @@ public void testNewMeasurePassesIfCqlLibraryNameStartsWithCapitalCharAndFollowed
     verifyNoMoreInteractions(measureRepository);
   }
 
-  @Test
-  public void testNewMeasureFailsWithInvalidModelType() throws Exception {
```
Comment (jkotanchik-SB) [inline src/test/java/cms/gov/madie/measure/resources/MeasureControllerMvcTest.java:376]: Moved lower in the file so that the CQL naming tests can be co-located.

**Review by adongare — APPROVED** [review]
